openpype/hosts/maya/plugins/publish/collect_review.py

```diff
@@ -139,3 +139,13 @@ def process(self, instance):
                         "filename": node.filename.get()
                     }
                 )
+
+        # Collect focal length.
+        data = {
```

Comment:
```suggestion
        instance.data["customData"] = []
        data = {
            "cameraFocalLength": cmds.getAttr(camera + ".focalLength")
        }
        instance.data["customData"].append(data)
        self.log.debug("camera focal length:{}".format(data))


```
or
```suggestion
        data = {
            "cameraFocalLength": cmds.getAttr(camera + ".focalLength")
        }
        instance.data["customData"] = data 
        self.log.debug("camera focal length:{}".format(data))


```
when I tried the code, it seems that it always looks at `instance.data["customData"] =data ` instead of `instance.data["customData"].update(data)`.